var/spack/repos/builtin/packages/nrm/package.py

```diff
@@ -14,6 +14,8 @@ class Nrm(PythonPackage):
     version("0.1.0", sha256="911a848042fa50ed216c818e0667bcd3e4219687eb5a35476b7313abe12106dc")
 
     depends_on("py-setuptools", type=("build"))
+    # pip 23.1 enforces to have a pyproject.toml
```

Comment:
I find this hard to believe. What was the error message?

Reply:
Actually I just tested it with another package without `pyproject.toml` and noticed that this does not seem to be the problem. I could not make sense out of the error message and when I googled I found somewhere this is due to missing pyproject.toml files. Seems that is wrong. Will try to find out what the actual problem is.

Reply:
Ok, I it seems that there was an error message that I overlooked:
```
    File "$spack/opt/spack/linux-fedora37-x86_64/gcc-12.3.1/python-3.8.17-bl3amrqv7
vomgl6dxgbbpnsllpcg7p6/lib/python3.8/zipfile.py", line 362, in __init__
      raise ValueError('ZIP does not support timestamps before 1980')
  ValueError: ZIP does not support timestamps before 1980
```

Reply:
I have no idea why this happens with py-pip@23.1 but not with py-pip@32.0.
I have exchanged the comment with a description of the error message. Is this enough?

Reply:
Yeah, that's good enough for now. Someone who really cares about nrm can fix it.